tools/harness-simulation/README.md

```diff
@@ -0,0 +1,26 @@
+# Test Harness on Simulation Environment Setup
```

Comment:
@Vyrastas , can you help get the current `README.md` reviewed?

Reply:
If we want to improve this for later, maybe open an issue for that and I can have someone from my team look at it when we have some bandwidth.

Also a general nit on the doc: add line breaks so no line is longer than 80 characters (rather than letting it word wrap) to make it easier to comment in specific areas for reviews.

Reply:
Unfortunately, the check "Build / pretty" does not allow line breaks inside a paragraph.
I also checked other markdown files in the repository, they do not obey the rule that no line is longer than 80 characters.

Reply:
No worries, thanks for checking.